.gitignore

```
@@ -10,7 +14,20 @@ install-sh
 missing
 test-driver
 config.h.in
-config.h.in~
+**/.deps/
+**/.dirstamp/
```

I think the `**/` at the start of this is redundant.

From the git docs:

> 
> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.
> 
> If there is a separator at the end of the pattern then the pattern will only match directories, otherwise the pattern can match both files and directories.
> 
> For example, a pattern doc/frotz/ matches doc/frotz directory, but not a/doc/frotz directory; however frotz/ matches frotz and a/frotz that is a directory (all paths are relative from the .gitignore file).
> 

```suggestion
.deps/
.dirstamp/
```